### ISSUE TITLE
Use of reserved keyword 'await'

### ISSUE THREAD

**Issue by matthiasg** [MatAtBread/nodent#70]
When trying to compile nodent into a browser package, tools like rollup complain that `arboriculture.js` is using reserved keyword `await` in line 2107 `function isHoistableFunction`.

It would be nice if this could be changed to some other name.

**Comment by matAtWork** [comment]
Interesting - `await` is only reserved (I think) in modules or within async functions in scripts. There's some talk of an issue with rollup at https://github.com/rollup/rollup/issues/564. But I've changed it anyway - it's not a problem for me - released in v3.0.9

**Comment by jdalton** [comment]
@matthiasg Would you post an example of using nodent with rollup?

**Comment by matthiasg** [comment]
I am currently not working in that area, but in about 2-3 weeks I will revisit and do that

**Comment by matthiasg** [comment]
@jdalton I just revisited this and it compiles fine now. As for your request posting an example I had initially read it as an example to reproduce the compile error. But I think now you meant an example with nodent as a rollup plugin. This would actually be a good idea.

Though I am not using nodent for that currently , I think I might just do it during our current development. We are extremely interested in providing async/await for our business scripts (easier to explain than promises for non developer users) the only reason we where waiting is usable source map support.

As for nodent for our own code we would certainly love that, but the source maps situation with webpack/rollup/chrome is far from ideal as is, thus we are hesitant to add additional complexity by using another layer of indirection.

I will post here again when I did an isolated nodent/rollup example
